### PR TITLE
Add BingTranslator#translate_array + small other changes

### PR DESCRIPTION
**Pull request by abrisse**

### CODE REVIEW

**Comment by abrisse** [comment]
FYI the branch does not pass travis tests because of the credentials.

File: lib/bing_translator.rb [hunk 1]
```diff
@@ -42,7 +42,22 @@ def translate(text, params = {})
...
+    # Important notice: param order makes sense in SOAP. Do not reorder or delete!
+    params = {
+      'texts'       => { 'arr:string' => texts },
+      'from'        => params[:from].to_s,
+      'to'          => params[:to].to_s,
+      'category'    => 'general',
+      'contentType' => params[:content_type] || 'text/plain'
+    }
+
+    array_wrap(result(:translate_array, params)[:translate_array_response]).map{|r| r[:translated_text]}
```
Comment (tsion) [inline lib/bing_translator.rb]: `array_wrap` seems kind of hacky. What else is the API returning besides `Array`s?

Otherwise, this PR looks good to me.

Comment (abrisse) [inline lib/bing_translator.rb]: A single entry if the texts array only contains 1 element. That's why I had to use `array_wrap` which is a copy of Array#wraps defined in Rails (see http://apidock.com/rails/Array/wrap/class).

**Comment by relrod** [comment]
@tsion should we merge this in?

**Comment by tsion** [comment]
@relrod Yeah, LGTM. We should bump the minor version and publish after merging.

**Pull request merged**